build.gradle

```diff
@@ -466,6 +466,10 @@ subprojects {
     if (shouldUseJUnit5) {
       useJUnitPlatform {
         includeTags "integration"
+        // KAFKA-14109
```

Comment:
We don't typically include jira references in the code like this (it pollutes the code and goes stale typically).

Reply:
Addressed the comment in the latest revision.